R/zchunk_L124.nonco2_unmgd_R_S_T_Y.R

```diff
@@ -43,80 +41,186 @@ module_emissions_L124.nonco2_unmgd_R_S_T_Y_DISABLED <- function(command, ...) {
+      standardize_iso(col = "Country") %>%
+      change_iso_code('rou', 'rom') %>%                                                              # Convert Romania iso code to pre-2002 value
+      left_join(iso_GCAM_regID, by = "iso") %>%
+      # There are a set of iso codes in the GFED data that don't exist in the GCAM region mapping. Remove those now.
+      na.omit() %>%
+      group_by(GCAM_region_ID, Non.CO2, year ) %>%
+      summarize(ForestFire = sum(ForestFire), Deforest = sum(Deforest) ) %>%                         # Aggregate emissions by region, gas, and year
+      mutate(PctForestFire = ForestFire / (ForestFire + Deforest)) %>%                               # Compute share of emissions from forest fires
+      # There are regions where GFED data is zero for both forest fires and deforestation, leading to NAs
+      # Assume those missing values are places with 100% forest fires since these are easier to model in GCAM
+      replace_na(list(PctForestFire = 1)) %>%
+      mutate(PctDeforest = 1 - PctForestFire) %>%                                                    # Compute share of emissions from deforestation
+      select(-ForestFire, -Deforest) ->
+      FireShares_R_G_Y
+
+    # Downscale regional forest burning emissions to GLU based on the share of land in each GLU
+    # Use GFED to separate into forest fires and deforestation, which have different drivers in GCAM
+    L124.LC_bm2_R_UnMgdFor_Yh_GLU_adj %>%
+      group_by(GCAM_region_ID, year) %>%
+      mutate(land_share = value / sum(value)) %>%                                                      # Compute share of regional forest area in each GLU
+      select(-value) %>%
+      # There are places with land area but no emissions and vice versa. Use an inner_join to only get places with both
+      inner_join(filter(EDGAR_history, sector == "forest"), by = c("GCAM_region_ID", "year")) %>%       # Map in EDGAR emissions information
+      mutate(value = value * land_share) %>%                                                           # Compute forest emissions from EDGAR totals and land shares
+      select(-sector, -land_share) %>%
+      left_join(FireShares_R_G_Y, by = c("GCAM_region_ID", "Non.CO2", "year")) %>%                     # Map in GFED fire shares
+      # Assume missing values mean 100% forest fires since these are easier to model in GCAM
+      replace_na(list(PctForestFire = 1)) %>%
+      replace_na(list(PctDeforest = 0)) %>%
+      mutate(ForestFire = value * PctForestFire) %>%                                                   # Compute forest fire emissions
+      mutate(Deforest = value * PctDeforest) %>%                                                       # Compute deforestation emissions
+      select(-value, -PctForestFire, -PctDeforest) %>%
+      gather(technology, value, -GCAM_region_ID, -GLU, -Land_Type, -Non.CO2, -year) ->
+      L124.nonco2_tg_R_forest_Y_GLU
+
+    # Compute global average deforestation emissions coefficients
```

Comment:
Suggest adding an additional comment explaining why this is done (I agree with method, but might not be clear to future readers of the code.)